---
layout: page
title: Research
permalink: /publications/
description: Current research interests
---
<br/>

> [Google Scholar Profile](https://scholar.google.com/citations?user=Z2G1YhoAAAAJ)


<br/>

### Selected publications

**Harrison Specht**, Ed Emmott, Aleksandra Petelski, R. Gray Huffman, David H. Perlman, Marco Serra, Peter Kharchenko, Toni Koller, Nikolai Slavov. "Single-cell mass spectrometry quantifies the emergence of macrophage heterogeneity" ***bioRxiv*** 2019. doi.org/10.1101/665307

<br/>

[![](/images/SCOPE2-ac.png)](https://scope2.slavovlab.net/)

> The fate and physiology of individual cells are controlled by protein interactions. Yet, our ability to quantitatively analyze proteins in single cells has remained limited. To overcome this barrier, we developed SCoPE2. It lowers cost and hands-on time by introducing automated and miniaturized sample preparation while substantially increasing quantitative accuracy. These advances enabled us to analyze the emergence of cellular heterogeneity as homogeneous monocytes differentiated into macrophage-like cells in the absence of polarizing cytokines. SCoPE2 quantified over 2,700 proteins in 1,018 single monocytes and macrophages in ten days of instrument time, and the quantified proteins allowed us to discern single cells by cell type. Furthermore, the data uncovered a continuous gradient of proteome states for the macrophage-like cells, suggesting that macrophage heterogeneity may emerge even in the absence of polarizing cytokines. Parallel measurements of transcripts by 10x Genomics scRNA-seq suggest that SCoPE2 samples 20-fold more copies per gene, thus supporting quantification with improved count statistics. Joint analysis of the data indicated that most genes had similar responses at the protein and RNA levels, though the responses of hundreds of genes differed. Our methodology lays the foundation for automated and quantitative single-cell analysis of proteins by mass-spectrometry.

<br/>

<hr>

<br/>

**Harrison Specht**, Nikolai Slavov. "Transformative Opportunities for Single-Cell Proteomics." ***Journal of Proteome Research.*** 2018. doi.org/10.1021/acs.jproteome.8b00257

<br/>

[![](/images/pr-2018-002576_0003.jpeg)](doi.org/10.1021/acs.jproteome.8b00257)

> Many pressing medical challenges, such as diagnosing disease, enhancing directed stem-cell differentiation, and classifying cancers, have long been hindered by limitations in our ability to quantify proteins in single cells. Mass spectrometry (MS) is poised to transcend these limitations by developing powerful methods to routinely quantify thousands of proteins and proteoforms across many thousands of single cells. We outline specific technological developments and ideas that can increase the sensitivity and throughput of single-cell MS by orders of magnitude and usher in this new age. These advances will transform medicine and ultimately contribute to understanding biological systems on an entirely new level.

<br/>

<hr>
